railties/lib/rails/api/task.rb

```diff
@@ -9,7 +10,10 @@ class Task < RDoc::Task
             README.rdoc
             lib/active_support/**/*.rb
           ),
-          exclude: "lib/active_support/vendor/*"
+          exclude: %w(
+            lib/active_support/i18n_railtie.rb
+            lib/active_support/xml_mini/*
+          )
```

Comment:
The previous exclusions are "vendor = not our code" here, and "generator template = not really code at all" below.

These new values seem much more selective, and content-aware. Are we sure these shouldn't be addressed by rdoc directives in the affected files?